src/js/wrap.cpp

```diff
@@ -1,7 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the Apache 2.0 License.
-#include "js/wrap.h"
-
+#ifndef DELETE_MTE
```

Comment:
What's this?

Reply:
It's been my local companion to avoid auto clang format move the following line in the middle of the other includes, which would break the compilation of this file. I am ashamed of it.

Reply:
I don't follow. `clang-format` treats "this file's header" as special, and puts it first, so `js/wrap.h` should always be first?

See eg:

```cpp
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the Apache 2.0 License.

#include "ccf/common_endpoint_registry.h"

#include "ccf/common_auth_policies.h"
#include "ccf/historical_queries_adapter.h"
```

But also, what breaks when it isn't first?

Reply:
I apparently still had some dodgy config in my clang format VSCode extension (assuming all files were named `nodestate.cpp`...). The compilation error is fixed by including `js/wrap.h` in `conv.cpp`. 

I said I was ashamed of it!